File: 프로그래머스/lv0/120921. 문자열 밀기/문자열 밀기.py
def solution(A, B):
    count = 0
    C = A
    if A==B:
        return 0
    for i in range(len(A)):
        C = C[-1]+C[0:-1]
        if C==B:
            count+=1
            break
        count+=1
    if count==len(A):
        return -1
    else:
        return count

File: 프로그래머스/lv0/120921. 문자열 밀기/문자열 밀기.js
function solution(A, B) {
    var count = 0;
    // 문자열의 길이만큼 반복하면서, 오른쪽으로 밀면서 count하기. 같아지면 반복 종료.
    for (var a = 0;a<A.length;a++){
        if(A===B){
            return count
        }
        A = A[A.length-1]+A.slice(0,A.length-1)
        count+=1
    }
    return -1;
}